PocketKit/Sources/PocketKit/ReaderSettings/ReaderSettingsView.swift

```diff
@@ -5,44 +5,101 @@
+                )
+                .accessibilityIdentifier("reader-settings-font-size-stepper")
+                if settings.isPremium {
+                    Stepper(
+                        Localization.Reader.Settings.lineHeightLabel,
+                        value: settings.$lineHeightScaleFactorIndex,
+                        in: settings.settingIndexRange,
+                        step: settings.settingIndexStep
+                    )
+                    .accessibilityIdentifier("reader-settings-line-height-stepper")
                     Stepper(
-                        Localization.fontSize,
-                        value: settings.$fontSizeAdjustment,
-                        in: Constants.allowedAdjustments,
-                        step: Constants.adjustmentStep
+                        Localization.Reader.Settings.marginsLabel,
+                        value: settings.$marginsIndex,
+                        in: settings.settingIndexRange,
+                        step: settings.settingIndexStep
                     )
-                    .accessibilityIdentifier("reader-settings-stepper")
+                    .accessibilityIdentifier("reader-settings-margins-stepper")
+                    Picker(Localization.Reader.Settings.alignmentLabel, selection: settings.$alignmentOption) {
+                        ForEach(ReaderSettings.AlignmentOption.allCases, id: \.rawValue) { alignment in
+                            Text(alignment.rawValue)
+                                .tag(alignment)
+                        }
+                        .navigationBarTitleDisplayMode(.inline)
+                    }
+                } else {
+                    SettingsRowButton(
+                        title: Localization.Reader.Settings.premiumUpsellLabel,
+                        titleStyle: .settings.row.active,
+                        leadingImageAsset: .premiumIcon,
+                        leadingTintColor: Color(.ui.teal2)
+                    ) {
+                        settings.presentPremiumUpgrade()
+                    }
+                    .sheet(
+                        isPresented: $settings.isPresentingPremiumUpgrade,
+                        onDismiss: {
+                            settings.trackPremiumDismissed(dismissReason: dismissReason)
+                            if dismissReason == .system {
+                                settings.isPresentingHooray = true
+                            }
+                        }
+                    ) {
+                        PremiumUpgradeView(dismissReason: self.$dismissReason, viewModel: settings.makePremiumUpgradeViewModel())
+                    }
+                    .task {
+                        settings.trackPremiumUpsellViewed()
+                    }
+                }
+            }
+            .sheet(isPresented: $settings.isPresentingHooray) {
+                PremiumUpgradeSuccessView()
+            }
+            if settings.isPremium {
```

Comment:
Should this be a premium-only setting? There are still two settings for free users.

Reply:
we can talk about this. The main reasoning behind this choice is that additional features (and this would include reset to defaults), are dedicated to premium users